### DISCUSSION TITLE
Polyline, bezier and gradient

### DISCUSSION THREAD

**Discussion by hpcolus** [zepp-health/zeppos-forum#225] [Developer Feedback]
This code for a polyline with a gradient only yields a line with a curve but no gradient.  The curve_style parameter which it is specific to API 2.1 works in the simulator but not the gradient.  Is this a bug?  
```
		onInit() {
			console.log('index page.js on init invoke')
			lineDataList = [
				{ x: 0, y: px(200) },
				{ x: px(100), y: px(10) },
				{ x: px(200), y: px(50) },
				{ x: px(300), y: px(50) },
				{ x: px(400), y: px(200) }
			  ]
			  polyline = hmUI.createWidget(hmUI.widget.GRADKIENT_POLYLINE, {
				x: 0,
				y: px(200),
				w: px(480),
				h: px(200),
				line_color: 0x00ff00,
				line_width: 1
			  })
			  // polyline.clear()
			  polyline.addLine({
				data: lineDataList,
				count: lineDataList.length,
				color_from: 0xff0000,
				color_to: 0x141414,
				curve_style: true
			  })
		  
			this.init_view()
		},

```
![image](https://github.com/zepp-health/zeppos-forum/assets/104551107/ad16516a-7871-4571-af67-f65df874c674)

**Comment by MyoungXUE** [comment]
Hi, thanks for your feedback, we are sure this is a bug and will fix it as soon as possible, in the meantime we will mark the relevant feature as unavailable on the documentation.
